pkg/web/templates/partials/secure_envelope/secure_envelope_detail.html

```diff
@@ -105,12 +115,14 @@ <h3 class="font-semibold">Virtual Asset Service Providers (Legal Persons)</h3>
         </tr>
       </table>
     </section>
-    <!-- <div class="mt-4">
+    <div class="divider"></div>
```

Comment:
This divider was added to visually break up the content inside the accordion for now until I can sort out how to implement the nested accordions (unless we decide to change the approach as we continue to discuss design updates).

Reply:
The divider is certainly very helpful for now!